app/views/subnets/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <thead>
     <tr>
       <th class="col-md-3"><%= sort :name, :as => s_("Subnet|Name") %></th>
-      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network") %></th>
+      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network Address") %></th>
       <th class="col-md-3"><%= _("Domains") %></th>
```

Comment:
@domcleal Update the English translation PO file, and re-submit a patch, please help to review. Thanks.